examples/07-Train-traditional-ML-models-using-the-Merlin-Models-API.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Add the names of the metrics to the notebook text as well (right after the "Implicit models can be..." sentence). It would also help to give a link to the definitions of these metrics, no matter how commonly known we think they are.</p><p> </p><p>Also, do we want to say anything about the significance or strength of the values of these metrics. They seem a bit low to indicate strong prediction performance in a real-life setting, but for the purpose of a demo notebook, they seem ok.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>metrics listed and definitions linked</p>

Reply:
<p>This looks great. The only minor correction I'd point out is with the word "acti" in the paragraphs that starts with "Essentially,...". I believe you meant to say "actions".</p>